src/demo/app/examples/data.service.ts

```diff
@@ -19,9 +19,9 @@ export interface Person {
 	providedIn: 'root',
 })
 export class DataService {
-	private _gitHubAccountsCache = new Map<string, []>();
+	private http = inject(HttpClient);
 
-	constructor(private http: HttpClient) {}
+	private _gitHubAccountsCache = new Map<string, []>();
```

Comment:
If you really don't want to let anybody use it I would recommend you to use `readonly` and `#` private property. Also for other injects.

Reply:
@muuvmuuv  Good idea, but I think this can be separate PR, not to deviate the purpose of PR.. 
Don't want clutter up things in single PR.